handlers/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
-# 'service reload' was not always working, needs complete reload
+
 - name: firewalld reload firewalld
-  command: firewall-cmd --complete-reload
```

Comment:
@eni23 any problems with this?

Reply:
It is just not what someone would expect.
Especially not in production environments.

from man: https://firewalld.org/documentation/man-pages/firewall-cmd.html

"Reload firewall completely, even netfilter kernel modules. This will most likely terminate active connections, because state information is lost."

If you like add it as a second notify handler and include a variable which will choose that notify handler in case.
But I would recommend, that this is not the default reload then.

Reply:
Yes of course. I just want to check back with @eni23 why this was introduced.